version.tf

```diff
@@ -1,12 +1,14 @@
 terraform {
-  required_version = ">= 1.3.0, <1.7.0"
-  # If your module requires any terraform providers, uncomment the "required_providers" section below and add all required providers.
-  # Each required provider's version should be a flexible range to future proof the module's usage with upcoming minor and patch versions.
-
-  #  required_providers {
-  #    ibm = {
-  #      source  = "IBM-Cloud/ibm"
-  #      version = ">= 1.49.0, < 2.0.0"
-  #    }
-  #  }
+  required_version = ">= 1.3.0, <1.6.0"
```

Comment:
We are starting to support and test with OpenTofu, so this can be set back to <1.7.0 to allow it (applies to all submodules and examples as well)